lib/modules/versioning/debian/common.ts

```diff
@@ -0,0 +1,66 @@
+import type { DistroInfo, DistroInfoRecordWithVersion } from '../distro';
+
+export class RollingReleasesData {
+  private ltsToVer = new Map<string, DistroInfoRecordWithVersion>();
+  private verToLts = new Map<string, DistroInfoRecordWithVersion>();
+  private distroInfo: DistroInfo;
+
+  constructor(distroInfo: DistroInfo) {
+    this.distroInfo = distroInfo;
+  }
+
+  getVersionByLts(input: string): string {
+    this.refresh();
+    const schedule = this.ltsToVer.get(input);
+    if (schedule) {
+      return schedule.version;
+    }
+    return input;
+  }
+
+  getLtsByVersion(input: string): string {
+    this.refresh();
+    const di = this.verToLts.get(input);
+    if (di) {
+      return di.series;
+    }
+    return input;
+  }
+
+  has(version: string): boolean {
+    this.refresh();
+    return this.ltsToVer.has(version);
+  }
+
+  schedule(version: string): DistroInfoRecordWithVersion | undefined {
+    this.refresh();
+    let schedule: DistroInfoRecordWithVersion | undefined = undefined;
+    if (this.verToLts.has(version)) {
+      schedule = this.verToLts.get(version);
+    }
+    if (this.ltsToVer.has(version)) {
+      schedule = this.ltsToVer.get(version);
+    }
+    return schedule;
+  }
+
+  private refresh(): void {
```

Comment:
This is fired on every method of this class, although this has time complexity of O(1). do we need to over complicate things, say with introducing some timestamp based refresh mechanism? 

previously it was fired only once based on the `maps` size, but then we are stuck in "time". 
therefore we have to refresh in some kind of interval.

Reply:
I don't see any benefit to save that info if we recompute it on every call anyways

Reply:
I only like to have on small test like this to verify date handling for unintentional changes in future by some other author who doesn't know about long running renovate instances and module scope caching.

```ts
mockDate('future');
expect(isStable('buster')).toBeFalse();
mockDate('past');
expect(isStable('buster')).toBeTrue();